docs/getting-started/install-configure.md

```diff
@@ -329,7 +329,7 @@ kubectl apply -f provider.yaml
 ### Install Alibaba Provider
 
 ```
-PACKAGE=crossplane/provider-alibaba:master
+PACKAGE=crossplane/provider-alibaba:v0.1.0
```

Comment:
This is in anticipation of the first official release of `provider-alibaba`. There has been one `v0.0.1` alpha release